azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -284,22 +296,25 @@ public Set<Integer> getContainersOfTerminatedFlows() {
   }
 
   /**
-   * Get the flows of EXECUTION_STOPPED status
+   * Get the flows recently (last 15 minutes) terminated: status in (execution_stopped, killed,
+   * killing, failed, failed_finishing, failed_succeeded, canceled)
    */
   @NotNull
-  Set<Integer> getExecutionStoppedFlows() {
-    Set<Integer> executionStoppedFlows = new HashSet<>();
-    if (this.executionStoppedMap.containsKey(Status.EXECUTION_STOPPED)) {
+  Set<Integer> getRecentTerminationFlows() {
```

Comment:
Suggest rename to `getRecentlyTerminatedFlows`

Reply:
sure